lib/registrant.rb

```diff
@@ -0,0 +1,22 @@
+class Registrant
+
+    attr_reader :name,
+                :age,
+                :permit,
+                :license_data
```

Comment:
Minor note on this attribute -- you are assigning allowing classes outside this class (specifically the Facility class) assign values to keys in the license_data directly (`license_data[:written] = true` for example)

For the purposes of this project, allowing this manipulation or using an `attr_accessor` is just fine. Generally, though, we want to stay away from allowing outside classes to mutate our data. Another alternative here is to create `setter_methods`. For example, if a registrant takes the written exam, you could create a method here called `pass_written_exam` and inside that method, you'd set `licence_data[:written] = true`. In this way, we can control the setting of this value to make sure it's not something random (like it won't end up being `license_data[:written] = yes` or something inconsistent). We could also add controls to this method, for example, if we got a score argument, we could only set the `true` value `if score > 80`. Using a setter method gives us more control over setting this value rather than letting outside code change it to whatever it wants. 

Again, this simple of a case is no problem. Just wanted to share this context for your learning beyond these projects! This discussion revolves around the theme of encapsulation (an OOP principle)